### PR TITLE
DOC-6566: Corda 4.12.2 release notes

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview OS: https://niamh.preview.docs.r3.com/en/platform/corda/4.12/community/release-notes.html#corda-open-source-edition-4122-release-notes
Preview ENT: https://niamh.preview.docs.r3.com/en/platform/corda/4.12/enterprise/release-notes-enterprise.html#corda-enterprise-edition-4122-release-notes

### CODE REVIEW

**Comment by adelel1** [comment]
Hi @AnnaK-R3 / @paulmoloneyr3 
I have updated the release notes and pushed a commit (I removed the new features section and added as fixed issues - as its a patch release). All the updates are in the commit.
I also added a third party component upgrades table, however it needs updating with the correct dependency name and name similar to what we did in original 4.12 release.
@chriscochrane Please see above, can you provide Anna/Paul with the info for the third party upgrades table, plus any missed. This is for OS and ENT. Similar to what we did for the initial release.

**Review by adelel1 — DISMISSED** [review]

File: content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -17,6 +17,82 @@ weight: 10
...
+
+4. Pre-4.12 transactions are verified in an external verifier process when encountered. This process does not, by default, include all third-party libraries that shipped with Corda 4.11 and earlier, nor does it have the `drivers` directory on the classpath. If your contracts in the ledger attachments depend on such third-party libraries or any contents from the `drivers` directory in Corda 4.11 or earlier, you can place the necessary JAR files in a directory called `legacy-jars` within the node directory. Any JARs in this directory will be added to the classpath of the external verifier. The TVU will assist you in identifying and verifying the resolution of such issues.
+
+### Fixed issues
+
+* There is no need for the external verifier to use the `legacy-contracts` folder anymore. The external verifier verifies pre-4.12 transactions and now solely uses the database to retrieve the contract attachments.
+* An open telemetry span has been added around the send to multiple parties and receive from multiple parties operations.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md:67]:
```suggestion
* An open telemetry span has been added around the Send to Multiple Parties and Receive from Multiple Parties operations.
```

File: content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -17,6 +17,82 @@ weight: 10
...
+* Previously, the transaction builder would log any failed verification attempts when trying to add missing dependencies. Now, these failed attempts are no longer logged if they occur while determining the missing dependencies.
+* This release contains AMQP serialisation performance improvements.
+* It is now possible to create two nodes whose X.500 names has the same O field value but different OU values when using the driver DSL for testing.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md:70]:
```suggestion
* It is now possible to create two nodes whose X.500 names have the same Organisation (O) field value but different Organisation Unit (OU) values when using the driver DSL for testing.
```

File: content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md [hunk 3]
```diff
@@ -17,6 +17,82 @@ weight: 10
...
+* There's no longer a memory leak when creating a series of mock networks for testing purposes.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.12/enterprise/release-notes-enterprise.md:71]:
```suggestion
* There is no longer a memory leak when creating a series of mock networks for testing purposes.
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.12/community/release-notes.md [hunk 4]
```diff
@@ -54,8 +54,8 @@ The following section describes the updated requirements for running Corda 4.12
 * An open telemetry span has been added around the send to multiple parties and receive from multiple parties operations.
 * Previously, the transaction builder would log any failed verification attempts when trying to add missing dependencies. Now, these failed attempts are no longer logged if they occur while determining the missing dependencies.
 * This release contains AMQP serialisation performance improvements.
-* It is now possible to create two nodes whose X.500 names has the same O field value but different OU values when using the driver DSL for testing.
-* There's no longer a memory leak when creating a series of mock networks for testing purposes.
+* It is now possible to create two nodes whose X.500 names have the same O field value but different OU values when using the driver DSL for testing.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.12/community/release-notes.md:57]:
```suggestion
* It is now possible to create two nodes whose X.500 names have the same Organisation (O) field value but different Organisation Unit (OU) values when using the driver DSL for testing.
```

**Review by paulmoloneyr3 — DISMISSED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**